styles/LexArcana.css

```diff
@@ -0,0 +1,236 @@
+    background-size: 32px;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-bottom: auto;
+}
+
+.system-LexArcana .rollcustom-icon { background-image: url("../styles/images/dices_ico_def.webp"); }
+.system-LexArcana .roll1d-icon { background-image: url("../styles/images/dices_ico_1d.webp"); }
+.system-LexArcana .roll2d-icon { background-image: url("../styles/images/dices_ico_2d.webp"); }
+.system-LexArcana .roll3d-icon { background-image: url("../styles/images/dices_ico_3d.webp"); }
+.system-LexArcana .delete-icon { background-image: url("../styles/images/trash_ico.webp"); }
+.system-LexArcana .add-icon { background-image: url("../styles/images/plus_ico.webp"); }
+
+.system-LexArcana input.peritia-value
+{
+    width: 32px;
+}
+
+.system-LexArcana button
+, .system-LexArcana .dialog-buttons button.default
+, .system-LexArcana .dialog-buttons button.yes
+, .system-LexArcana .dialog-buttons button.no
+, .system-LexArcana .form-group button
+, .system-LexArcana select
+, .system-LexArcana textarea
+, .system-LexArcana form input
+, .system-LexArcana .directory-footer
+, .system-LexArcana .sheet-body
+, .system-LexArcana .avatar
+, .system-LexArcana .window-content .sheet-header .profile
+, .dialog .dialog-buttons button
+, input[type='text']
+{
+    color: #000000;
+    background-color: #ffffff88;
+    text-align: center;
+}
+
+.system-LexArcana .peritia-name { font-weight: bold; }
+
+/**********************************************************
+ * End of Character sheets
+ **********************************************************/
+
+.system-LexArcana .chat-message .message-header
+{
+    color: #d70000;
+}
+
+.system-LexArcana .chat-message.whisper
+{
+    background-color: #00000088;
+}
+
+.system-LexArcana .chat-message.whisper .message-header
+{
+    background-color: #00000088;
+    color: #ffd700;
+}
+
+.system-LexArcana .form-header
+, .system-LexArcana #player-config .window-content form .form-group label
+, .system-LexArcana .sidebar-tab p
+, #module-management .package-list .package-title
+, #module-management .notes
+{
+    color: #ffffff;
+}
+
+.system-LexArcana .form-group .units
+{
+    color: #ffd700;
+}
+
+.system-LexArcana .window-content .sheet-header .profile
+, .system-LexArcana .avatar
+, .system-LexArcana #player-config .window-content form .form-group
+{
+    border-radius: 1em;
+}
+
+.system-LexArcana .window-content .sheet-header .attribute-value input
+{
+    width: 40px;
+}
+
+.system-LexArcana .tabs
+{
+    margin-top: 5px;
+    border-top-left-radius: 1em;
+    border-top-right-radius: 1em;
+}
+
+.system-LexArcana .tabs .item
+{
+    text-shadow: 0px 0px 4px #00000088;
+}
+
+.system-LexArcana .tabs .active
+{
+    text-shadow: 0px 0px 4px #ffd700ff;
+}
+
+.system-LexArcana .biography, .system-LexArcana .editor, .system-LexArcana .editor-content, .system-LexArcana .tox-edit-area
+{
+    height: 100%;
+}
+
+.editor-content
+, .tox
+{
+    padding: 5px;
+	background-color: #e0cfaa;
```

Comment:
indentation

Reply:
not in visual studio code nor notepad++ 🤔